receiver/awsecscontainermetricsreceiver/go.mod

```diff
@@ -4,7 +4,9 @@ go 1.14
 
 require (
 	github.com/census-instrumentation/opencensus-proto v0.3.0
+	github.com/golang/protobuf v1.4.2
 	github.com/stretchr/testify v1.6.1
 	go.opentelemetry.io/collector v0.10.1-0.20200917170114-639b9a80ed46
 	go.uber.org/zap v1.16.0
+	google.golang.org/protobuf v1.25.0
```

Comment:
Looks like these are from importing`github.com/golang/protobuf/ptypes/timestamp` and `google.golang.org/protobuf/types/known/timestamppb`. Do you need both? Could probably just use `timestamppb` in both places.

Reply:
Updated.